### PR TITLE
(WIP - not working yet) Attempt to fix libm linking issue on linux (vercel/netlify etc)

### PR DESCRIPTION
**Pull request by JasoonS**
fixes: #97

I'm super new to esy - not sure what I'm doing wrong, but hopefully most of the way there!

I took inspiration from [discord](https://discord.com/channels/235176658175262720/235200837608144898) and [this PR](https://github.com/zth/reason-relay/pull/158/files?file-filters%5B%5D=.json&file-filters%5B%5D=No+extension) doing something similar for reason-relay.

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -58,11 +58,18 @@ jobs:
...
+        if: matrix.os != 'ubuntu-latest'
         run: |
           cd lib
           esy build
 
+      - name: Build ppx (linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cd lib
+          esy dune build -p '#{self.name}' --profile release-static
```
Comment (JasoonS) [inline .github/workflows/pipeline.yml:71]: I couldn't get this to work as `esy linux-release-static`, I'm not sure why - it was added to the package.json as described here: https://esy.sh/docs/en/configuration.html#project-specific-commands

The command also fails on my ubuntu laptop, so I'm not sure what is wrong.
image

**Comment by JasoonS** [comment]
image

I'm really not sure what is wrong with the command. I get this when I run `esy release-static`

**Comment by alexfedoseev** [comment]
Hey, thanks for the effort! I made some progress in #99 but it still fails with odd error. I'll comeback to it some time this week.

**Comment by JasoonS** [comment]
Good luck!

**Comment by JasoonS** [comment]
(I'm on pop-os (which is basically a flavour of ubuntu) 20.04 if that is relevant) - I didn't try run it in vercel where I got the initial error yet.

**Comment by alexfedoseev** [comment]
Yeah, I was able to reproduce it locally in a docker container. It means there is still an issue with the way ppx binary is built. I'm out of ideas but I will ask around on OCaml forums.

**Pull request closed without merging**